README.md

```diff
@@ -213,42 +213,31 @@ services:
         # group and group-id are specified, the group's GID must match group-id.
         group-id: <gid>
 
-        # (Optional) If the service exits, perform this action: either restart
-        # the service after a backoff wait, exit the Pebble server, or simply
-        # ignore it (log the failure and transition the service to the
-        # "stopped" status). Default is "restart".
-        #
-        # If on-failure is set and the service's exit code is nonzero,
-        # on-failure takes precedence; if on-success is set and the service's
-        # exit code is zero, on-success takes precedence.
-        on-exit: restart | exit | ignore
-
-        # (Optional) If the service exits with a nonzero exit code, override
-        # on-exit and perform this action (see on-exit for details).
-        on-failure: restart | exit | ignore
+        # (Optional) If the service exits with a zero exit code, perform
+        # this action: either restart the service after a backoff wait, halt
+        # (terminate the Pebble server), or simply ignore it (log the failure
+        # and transition the service to the "stopped" status). Default is
```

Comment:
Sorry for missing this earlier: I think we're not yet exposing (and probably should not) the "stopped" status that is used by the internal state machine, but rather showing this up as "inactive".  We need to be a bit careful in terms of consistency between the internal logic and what we have in the user interface. The sentence also seems slightly broken up, when perhaps we could make it more linear. Giving it a try:
```
# (Optional) The on-success option defines what happens when the service exits with
# a zero exit code. Possible values are: 'restart' (default) which restarts the service after
# the backoff time, 'halt' which stops Pebble and exits, and 'ignore' which does nothing
# in that situation.

Reply:
Thanks. Updated to basically your wording with a couple of tweaks.